exercises/practice/bank-account/.meta/example.cpp

```diff
@@ -0,0 +1,54 @@
+#include "example.h"
+
+#include <stdexcept>
+
+void Bankaccount::open() {
+    if (open_) {
+        throw std::logic_error{"account already open"};
+    }
+    open_ = true;
+    balance_ = 0;
+}
+
+void Bankaccount::deposit(int amount) {
+    std::lock_guard<std::mutex> guard(mutex_);
```

Comment:
Since C++17 this can be written as `std::lock_guard guard(mutex_);`, the template parameter will be deduced by the compiler.